FAQ.md

```diff
@@ -0,0 +1,82 @@
+# FAQ
```

Comment:
Assume it's possible someone comes to this page without having read anything else.

Additional possible FAQs:

+ Where do I learn more about Triton Object Storage (Manta)? -- this can be served with a link to the docs
+ What are some other projects using the Java-Manta SDK? -- direct people back to the main README list

Do we have any other notable SDKs and would someone using the Java-Manta SDK care?